.flake8

```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length = 120
+extend-ignore = E203
+per-file-ignores =
+    # imported but unused
+    __init__.py: F401
+    main.py: F403, F405
```

Comment:
Let's resolve these instead of ignoring them

Reply:
And add lint workflow in this PR :mhm:

Reply:
> Let's resolve these instead of ignoring them

`__init__.py` can't be resolved, but sure .-.